src/app/components/tabs/index.html

```diff
@@ -1,15 +1,292 @@
+				</div>
+			</div>
+		</div>
+		
+	<h3>Don't use when</h3>
+		<div class="two-col-right-image  sky-margin-stacked-content-section">
+			<div class="two-col-text">
+				<p>You already have a set of tabs on the page. Try changing the workflow rather than using multiple sets of tabs.</p>
+			</div>
+			<div class="two-col-image">
+				<div class="screenshot screenshot-danger sky-margin-stacked-content-section">
+					<img class="bordered" src="~/assets/img/guidelines/tab/dont-double-tab-horz.png" />
+					<div class="caption sky-text-danger">Don't use multiple sets of tabs on a single page.</div>
+				</div>
+				<div class="screenshot screenshot-danger">
+					<img class="bordered" src="~/assets/img/guidelines/tab/dont-double-tab-vert.png" />
+					<div class="caption sky-text-danger">Don't combine vertical tabs with horizontal tabs.</div>
+				</div>
+			</div>
+		</div>
+    
+    <div class="two-col-right-image sky-margin-stacked-separate">
+			<div class="two-col-text">
+				<p>The tabs are likely to overflow the available horizontal space, which will lead to the collapsed tab dropdown and a large amount of unused space in the tab bar. Use <a routerLink="/components/vertical-tabs">vertical tabs</a> instead.</p>
+			</div>
+			<div class="two-col-image">
+				<div class="screenshot screenshot-danger">
+					<img class="bordered" src="~/assets/img/guidelines/tab/dont-overflow.png" />
+					<div class="caption sky-text-danger">Don't use horizontal tabs when they are likely to overflow.</div>
+				</div>
+			</div>
+		</div>
+		
+	</sky-demo-page-content>
+	
+	<sky-demo-page-content sectionHeading="Anatomy">
+		<div class="two-col-right-image sky-margin-stacked-separate">
+			<div class="two-col-text">
+				<div class="anatomy-list">
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">1</div><div class="anatomy-description">Selected tab</div></div>
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">2</div><div class="anatomy-description">Unselected tab</div></div>
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">3</div><div class="anatomy-description">Tab border</div></div>
+					<div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">4</div><div class="anatomy-description">Tab count (optional)</div></div>
+					<div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">5</div><div class="anatomy-description">Close tab button (optional)</div></div>
+					<div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">6</div><div class="anatomy-description">Open tab button (optional)</div></div>
+				</div>
+			</div>
+			<div class="two-col-image">
+				<div class="screenshot">
+					<img class="bordered" srcset="~/assets/img/guidelines/tab/anatomy.png 1x, ~/assets/img/guidelines/tab/anatomy_2x.png 2x" src="~/assets/img/guidelines/tab/anatomy.png" />
```

Comment:
In this graphic, it looks slightly odd to me that we use small lines with items 2 and 3 but none the others. I defer to your design expertise, but I wonder if we really need the lines here. I think the placement of 2 makes it clear what it is pointing to without the line, and I think it would be more effective to move 3 down to touch the border instead of using a line to connect it to the border (similar to how #2 touches the highlight border in the corresponding image in the Anatomy section of the popover docs).

Reply:
The general approach we are taking is to only use the pointer lines where necessary, even if it leads to mixed instances with and without. In this case there is some awkwardness of overlaying the dots on the dark blue elements making the pointers necessary on 3.

Reply:
Makes sense. Thanks.